packages/utilities/src/DelayedRender.tsx

```diff
@@ -49,11 +49,13 @@ export class DelayedRender extends React.Component<IDelayedRenderProps, IDelayed
 
   public componentDidMount(): void {
     let { delay } = this.props;
-    this._timeoutId = setTimeout(() => {
+    this._timeoutId = (setTimeout(() => {
       this.setState({
         isRendered: true
       });
-    }, delay);
+      // This cast has to be done so that Fluent can take @uifabric/utilities as a dependency given diferent transpilations between TS to
+      // ES5 and babel to ES6-like.
+    }, delay) as unknown) as number;
```

Comment:
I don't see how it's related to Babel as you're fixing types there as I believe you have there this issue:
https://stackoverflow.com/questions/45802988/typescript-use-correct-version-of-settimeout-node-vs-window

Can you please clarify?

Reply:
I am having that problem, I thought it was babel enforcing this difference but I might have been wrong. I'll use the fix in the link you sent me and delete the comment, thanks!